public/get-started/index.html

```diff
@@ -81,20 +81,76 @@ <h1 class="title">Get Started</h1>
       <div class="section text-center landing-section">
         <div class="container">
 
-          <p style="font-size: 1.45em" class="title">Step 1</p>
+          
           <div class="row">
             <div class="col-12">
-                <p class = "text-center">Head over to our admin panel <a href="#" target="_blank"> by clicking here</a></p>
+              <p class="title stepTitle">Step 1</p>
+                <p class = "text-center">Head over to our admin panel at <a href="https://staging.cohort.rocks/admin" target="_blank"> https://staging.cohort.rocks/admin by clicking here</a></p>
```

Comment:
On this page I think it would be good to left-align the body text to help make the instructions more readable.